lib/ioc-rc

```diff
@@ -555,7 +557,8 @@ __rc_jails () {
         local boot_order=$(sort -n $boot_list)
         local shutdown_order=$(sort -rn $boot_list)
     else
-        __info "no jails found with property boot=on, exiting.."
+        __info "no jails found with property boot=on or
+        with jail_hostid: $hostid on this host, exiting.."
```

Comment:
Do you want both lines moved over ?

INFO: no jails found with property boot=on or
        with jail_hostid: deffca19-ceff-e91c-ceff-e91cd3ff6a1a on this
host, exiting..

Op 14/09/15 om 23:47 schreef Brandon Schneider:
>
> In lib/ioc-rc
> <https://github.com/iocage/iocage/pull/222#discussion_r39451751>:
>
> > @@ -555,7 +557,8 @@ __rc_jails () {
> >          local boot_order=$(sort -n $boot_list)
> >          local shutdown_order=$(sort -rn $boot_list)
> >      else
> > -        __info "no jails found with property boot=on, exiting.."
> > +        __info "no jails found with property boot=on or
> > +        with jail_hostid: $hostid on this host, exiting.."
>
> Please move this over 4 spaces.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iocage/iocage/pull/222/files#r39451751>.
>